app/src/services/transformService.js

```diff
@@ -7,6 +7,19 @@ moment.createFromInputFallback = function (config) {
 };
 
 const transformService = {
+  stringToDate: s => {
+    if (s && s.trim().length) {
+      return moment(s).toDate();
+    }
+    return null;
+  },
+
+  dateToString: d => {
+    if (d) {
+      return moment(d).format('YYYY-MM-DD');
+    }
+    return '';
+  },
```

Comment:
These two functions could be rewritten as ternaries and onelined and I believe it would still retain clarity.

```
stringToDate: s => (s && s.trim().length) ? moment(s).toDate() : null,
dateToString: d => d ? moment(d).format('YYYY-MM-DD') : ''
```